packages/core/src/CoreAPI.ts

```diff
@@ -0,0 +1,88 @@
+import { IdenityRegistry } from '@tangleid/did';
+import { signRsaSignature, verifyRsaSignature } from '@tangleid/jsonld';
+import { DocumentLoader } from './jsonld/documentLoader';
+
+import { IriProvider, Seed, Did, PublicKeyMeta, PrivateKeyPem, PublicKeyPem } from '../../types';
```

Comment:
Again. Send another pull request for revising `IriProvider`. Something like `TangleEndpoint` might be better.

Reply:
Revised at #172 .